site/docs/v1/tech/developer-guide/api-gateways/kong-gateway.adoc

```diff
@@ -0,0 +1,190 @@
+
+image::api-gateways/kong-gateway/kong-gateway-create-user.png[FusionAuth User Creation, width=1200,role=bottom-cropped]
+
+After saving this user, click [uielement]#Manage# and go to the [breadcrumb]#Registrations# tab. Click [uielement]#Add Registration# to register the user to your application.
+
+image::api-gateways/kong-gateway/kong-gateway-user-registration.png[FusionAuth User Registration, width=1200]
+
+== Add a Consumer in Kong Gateway
+
+Now that FusionAuth has all the requisite information to create a JWT, you can configure your Kong Gateway and JWT plugin to provide access based on valid tokens created by FusionAuth.
+
+Execute the following command in your terminal to add a consumer to Kong.
+
+[source,shell,title="Creating a consumer"]
+----
+curl -i -X POST http://localhost:8001/consumers \
+  --data "username=fusionauth"
+----
+
+This command creates a Kong consumer with the username `fusionauth`. This consumer will be linked to the authentication information that you supply to it, namely the [field]#Public key# and [field]#Issuer# name from FusionAuth. This consumer uses the information provided to verify any JWT used to access the service. You can enable this with the following command:
+
+[source,shell,title="Supply key to consumer"]
+----
+curl -i -X POST http://localhost:8001/consumers/<CONSUMER_USERNAME>/jwt \
+  -F "algorithm=RS256" \
+  -F "rsa_public_key=@./<PUBLIC_KEY_FILENAME>" \
+  -F "key=<YOUR_FUSIONAUTH_URL>"
+----
+
+Here, `YOUR_FUSIONAUTH_URL` should exactly match the [field]#Issuer# field from [breadcrumb]#Tenants -> Your Tenant#, `CONSUMER_USERNAME` should match the [field]#username# that you supplied to the consumer in the prior step, and `PUBLIC_KEY_FILENAME` should match the name of the file that you saved the public key to, from the [breadcrumb]#Key Master# page.
+
+Using the example names from this guide, the command will appear as follows:
+
+[source,shell,title="Supply key to consumer with example values"]
+----
+curl -i -X POST http://localhost:8001/consumers/fusionauth/jwt \
+  -F "algorithm=RS256" \
+  -F "rsa_public_key=@./public.pem" \
+  -F "key=https://local.fusionauth.io"
+----
+
+== Creating and Using the Service with a JWT
+
+You can now generate a test JWT to access your Kong Gateway service using FusionAuth's login API. Execute the following command in your terminal.
+
+[source,shell,title="Get JWT token from FusionAuth"]
+----
+curl --location --request POST '<YOUR_FUSIONAUTH_URL>/api/login' \
+--header 'Authorization: <API_KEY>' \
+--header 'Content-Type: application/json' \
+--data-raw '  {
+    "loginId": "<USER_EMAIL>",
+    "password": "<USER_PASSWORD>",
+    "applicationId": "<APPLICATION_ID>",
+    "noJWT" : false
+  }'
+----
+
+Here, `YOUR_FUSIONAUTH_URL` is the [field]#Issuer# name, and `API_KEY` is the key you set up on the [breadcrumb]#Settings -> API Keys# page.
+
```

Comment:
A bit weird to have the picture here, as the user isn't on that screen any more. please move this picture up to where you create the API key.